### DISCUSSION TITLE
I can't load on desktop browsers scenes larger than 17.5 Mb

### DISCUSSION THREAD

**Discussion by APercy** [mozilla/hubs#5179] [Hubs Cloud Questions]
I imported several scenes available in hubs to my own server, but I can only load in rooms scenes smaller than 17.5Mb in desktop. But I got normal access on my iphone 7. What could it be?

**Comment by pattersonbl2** [comment]
Can you include the errors you are getting?

**Reply by APercy** [reply]
"DevTools failed to load source map: Could not load content for https://meta.evtest.info/sketchfab-zip.worker-f9825849872abe47c587.js.map: Fetch through target failed: Target not supported; Fallback: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE"

gltf-model-plus.js:44 Releasing uncached gltf https://meta.evtest.info/files/0b315793-3842-4429-86fe-05c80bee0801.bin
release @ gltf-model-plus.js:44
(anonymous) @ gltf-model-plus.js:849
m @ scene-9426caa4f82cac0242d9.js:31
s @ scene-9426caa4f82cac0242d9.js:31
gltf-model-plus.js:850 Failed to load glTF model ProgressEvent {isTrusted: true, lengthComputable: false, loaded: 0, total: 0, type: 'error', …} n {el: a-entity, id: undefined, attrName: 'gltf-model-plus', evtDetail: {…}, initialized: true, …}
(anonymous) @ gltf-model-plus.js:850
m @ scene-9426caa4f82cac0242d9.js:31
s @ scene-9426caa4f82cac0242d9.js:31
/files/0b315793-3842-4429-86fe-05c80bee0801.bin:1 Failed to load resource: net::ERR_CONNECTION_CLOSED
